src/keyboard.js

```diff
@@ -82,14 +83,34 @@ export function getCode( key ) {
  */
 export function parseKeystroke( keystroke ) {
 	if ( typeof keystroke == 'string' ) {
-		keystroke = keystroke.split( /\s*\+\s*/ );
+		keystroke = splitKeystrokeText( keystroke );
 	}
 
 	return keystroke
 		.map( key => ( typeof key == 'string' ) ? getCode( key ) : key )
 		.reduce( ( key, sum ) => sum + key, 0 );
 }
 
+/**
+ * It translates any keystroke string text like `"CTRL+A"` to an
+ * environment–specific keystroke, i.e. `"⌘A"` on Mac OSX.
+ *
+ * @method utils.keyboard.getEnvKeystrokeText
+ * @param {String} keystroke Keystroke text.
+ * @returns {String} Keystroke text specific for the environment.
+ */
+export function getEnvKeystrokeText( keystroke ) {
+	const split = splitKeystrokeText( keystroke );
+
+	if ( env.mac ) {
+		if ( split[ 0 ].toLowerCase() == 'ctrl' ) {
```

Comment:
`SHIFT+CTRL+A` is not going to work. We need to either document that CTRL must always be first or handle this case. I guess that documenting a standard way how keystrokes should be defined will be best. This can be done in this module for simplicity – e.g. as a new type `KeystrokeText`.